### DISCUSSION TITLE
Add fetched m3u8 data instead of url

### DISCUSSION THREAD

**Discussion by shubham7101** [vidstack/player#1040] [Q&A] [answered]
Please anyone can help me.

I am getting fetched m3u8 data instead of url. So how can I implemented this in vidstack.

**Comment by mihar-22** [comment]
I don't understand the question. Can you show an example of what you're trying to do, any errors, and what you expect to happen?

**Reply by shubham7101** [reply]
I mean the url from m3u8 is like this
https://test-streams.mux.dev/x36xhzz/x36xhzz.m3u8

When I curl using command,
`curl https://test-streams.mux.dev/x36xhzz/x36xhzz.m3u8`

I get output
`#EXTM3U
#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=2149280,CODECS="mp4a.40.2,avc1.64001f",RESOLUTION=1280x720,NAME="720"
url_0/193039199_mp4_h264_aac_hd_7.m3u8
#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=246440,CODECS="mp4a.40.5,avc1.42000d",RESOLUTION=320x184,NAME="240"
url_2/193039199_mp4_h264_aac_ld_7.m3u8
#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=460560,CODECS="mp4a.40.5,avc1.420016",RESOLUTION=512x288,NAME="380"
url_4/193039199_mp4_h264_aac_7.m3u8
#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=836280,CODECS="mp4a.40.2,avc1.64001f",RESOLUTION=848x480,NAME="480"
url_6/193039199_mp4_h264_aac_hq_7.m3u8
#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=6221600,CODECS="mp4a.40.2,avc1.640028",RESOLUTION=1920x1080,NAME="1080"
url_8/193039199_mp4_h264_aac_fhd_7.m3u8`


So is it possible to load vidstack player using curl data instead of m3u8 link.

**Reply by mihar-22** [reply, marked as answer]
Why? You can just provide the the initial URL to the player. How do you imagine providing the data and why?

**Reply by shubham7101** [reply]
Bcz I thinked that it will solve cors error. But I was wrong sorry for bothering you 😔.


Is it possible to add multiple m3u8 link
Bcz I am receiving 
`{ 
"sources":[
  {
    "url":"",
    "lang": "English"
  },
  {
    "url":"",
    "lang":"hindi"
  }
]
}`
